utility/src/main/java/org/oppia/android/util/logging/firebase/LogReportingModule.kt

```diff
@@ -18,4 +19,11 @@ class LogReportingModule {
   @Provides
   @Singleton
   fun provideEventLogger(factory: FirebaseEventLogger.Factory): EventLogger = factory.create()
+
+  @Provides
+  @Singleton
+  fun providePerformanceMetricsEventLogger(
```

Comment:
Any test to verify this binding?

Reply:
I suggest adding an explicit LogReportingModuleTest (see other module tests for an idea)--they're generally quick to setup, and are useful for directly verifying the exact binding.